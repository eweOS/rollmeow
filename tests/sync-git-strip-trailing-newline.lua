return {
	evalDownstream  = function(x) return "11.45.14"; end,
	cachePath       = "/tmp/rollmeow-test.cache.lua",
	packages        = {
		["mvim"] = {
			gitrepo	= "https://github.com/ziyao233/mvim.git",
			-- Matches nothing if there's a trailing newline
			regex   = "HEAD$",
		};
	},
};
