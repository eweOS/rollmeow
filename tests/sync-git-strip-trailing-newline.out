mvim: upstream HEAD | downstream 11.45.14
